sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -1791,7 +1840,11 @@ SUBROUTINE MDLFLD
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=1,IM
+                 IF(EGRID1(I,J)<spval.and.EGRID2(I,J)<spval)THEN
```

Comment:
I think you should just use

IF(EGRID3(I,J)<spval)THEN

Reply:
will do

Reply:
done